Resources/views/snippets/pkgShop/shopBasket/shopBasketMiniBasket.html.twig

```diff
@@ -24,7 +24,7 @@
     <div class="shopBasketMiniBasketBox">
         <div class="shopBasketMiniBasketBoxBoxInner">
             <div class="shopBasketMiniBasketBoxBoxSpacerRight">
-                <a class="shopBasketMiniBasketBoxBoxSpacerLeft teaser" href="{{sBasketURL|raw}}" title="{{"chameleon_system_chameleon_shop_theme.minibasket.action_show_basket_title"|trans}}">
```

Comment:
Does the sBasketUrl still exist?

If so the second "basketUrl" should have a name that hints what is corrected in it.

basketUrlWithoutX (?)

Reply:
Is the sBasketUrl value always wrong (or the caching of it)?

Reply:
`sBasketUrl` has the correct value when rendered uncached, but the caching of the whole template will lead to it being cached incorrectly. In the `chameleon-shop` change we opted to leave the template variable in place in order to not completely break projects - but you are right, I should add a warning message for the manual changes required in project templates